src/schema/columns/type.yaml

```diff
@@ -0,0 +1,35 @@
+---
+name: type
+description: |
+  Type of channel; MUST use the channel types listed below.
+  Note that the type MUST be in upper-case.
+type: string
+enum:
```

Comment:
Am I correct that you are pulling all the available types for EEG, MEG and iEEG ?

If I remember MEG has some that might not exist for EEG for example.

From the macros in the specs I see that you are not using this to generate the channel type tables. Am I getting this right?

Reply:
That's true. Unfortunately, we don't have any way to account for that just yet. I guess I could do an anyOf enum, though. That would separate the different options out a little bit.

Yeah, the channel tables aren't being generated from these files yet.

Reply:
Okidoki. Thanks for the clarification. :smiley: